README.md

```diff
@@ -41,6 +41,13 @@ You can run the controller locally on your machine by executing `make start`.
 
 Static code checks and tests can be executed by running `make verify`. We are using Go modules for Golang package dependency management and [Ginkgo](https://github.com/onsi/ginkgo)/[Gomega](https://github.com/onsi/gomega) for testing.
 
+## Caveats
+You can use all available disks on your Equinix instance, but only under 
+certain conditions:
+* You must use Flatcar
+* You must have a homogenous worker pool
```

Comment:
you should specifiy what homogeneous means.
I think, all worker pools must have the same image and container runtime